generators/app/utils.js

```diff
@@ -27,9 +24,8 @@ exports.flattenPrompts = prompts =>
       const generateNewPrompt = (prompt, positive, hasSelected = false) => ({
         ...prompt,
         when: answers => {
-          if (hasSelected && answers[actualPrompt.name] && answers[actualPrompt.name][hasSelected]) {
+          if (hasSelected && answers[actualPrompt.name] && answers[actualPrompt.name][hasSelected])
```

Comment:
Up to the possible extent, single statement ifs should be placed within the same line without using { }. But this isn't in the same line

Reply:
Ditto